activesupport/CHANGELOG.md

```diff
@@ -530,4 +530,17 @@
 
     *Jean Boussier*
 
+*   New methods to check if current date is start of month or end of month, returns boolean.
```

Comment:
```suggestion
* Add `DateAndTime#is_start_of_month?`, `DateAndTime#is_beginning_of_month?`, and `DateAndTime#is_end_of_month?` to check if current date is start of month or end of month, returns boolean.

```

I could be wrong about the class/module there, but we generally want to specify _which_ methods are being added in the changelog.

Also, there should be a new line between paragraphs.